students/jarosz_natalia/lesson_05/age_calculator.py

```diff
@@ -0,0 +1,25 @@
+def average_age(list_of_people):
+    adult_numbers = 0
+    adult_age_summary = 0
+    child_numbers = 0
+
+    for el in list_of_people:
```

Comment:
instead of cast el to integer twice in line 7 and line 8 better is cast el variable here or just declare in the line 19 just ```list_of_people = [2, 0, 58, 38]```